pkg/client/rootd/session.go

```diff
@@ -859,20 +742,10 @@ func (s *Session) stop(c context.Context) {
 	<-cc.Done()
 	atomic.StoreInt32(&s.closing, 2)
 
-	if s.stack != nil {
-		s.stack.Close()
-	}
-
-	cc = dcontext.WithoutCancel(c)
-	for _, np := range s.curStaticRoutes {
-		err := np.RemoveStatic(cc)
-		if err != nil {
-			dlog.Warnf(c, "error removing route %s: %v", np, err)
-		}
-	}
-	if s.dev != nil {
-		if err := s.dev.Close(); err != nil {
-			dlog.Errorf(c, "unable to close %s: %v", s.dev.Name(), err)
+	if s.tunVif != nil {
+		cc = dcontext.WithoutCancel(c)
```

Comment:
```suggestion
		cc, cancel := context.WithTimeout(dcontext.WithoutCancel(c), 1*time.Second)
		defer cancel()
```
to ensure that this never hangs.